ang/civicase.ang.php

```diff
@@ -232,12 +232,34 @@
 $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
 $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
 
-return array(
-  'js' => array(
+if ( ! function_exists('glob_recursive')) {
```

Comment:
Sorry, but seems like we have left a space after `!`. Is it some standard in PHP ?

Reply:
No, its a mistake, fixed. Thanks for noticing.